### DISCUSSION TITLE
Which tokenization technique is employed by BM25?

### DISCUSSION THREAD

**Discussion by lxx1220** [castorini/pyserini#1877] [Q&A]

**Comment by lintool** [comment]
Depends on language. For English: `DefaultEnglishAnalyzer` https://github.com/castorini/anserini/blob/master/src/main/java/io/anserini/index/IndexCollection.java#L62

For other languages: https://github.com/castorini/anserini/blob/master/src/main/java/io/anserini/index/IndexCollection.java#L213
